examples/default/README.md

```diff
@@ -1,7 +1,6 @@
-# Secret Manager and IAM Secrets Engine setup
+# Default example
 
-This example creates the following things:
-
-- A resource group (optional)
-- A Secrets Manager instance
-- An IAM Secrets engine
+This example uses module's default variable values. The following resources are provisioned by this example:
```

Comment:
A simple example that shows how to provision the Secrets manager IAM credentials engine.

The following resources are provisioned by this example:
- A new resource group, if an existing one is not passed in.
- An IBM Secrets manager instance
- The Secrets manager IAM credentials engine

Reply:
> A simple example that shows how to provision the Secrets manager IAM credentials engine.
> 
> The following resources are provisioned by this example:
> 
> * A new resource group, if an existing one is not passed in.
> * An IBM Secrets manager instance
> * The Secrets manager IAM credentials engine

Capitalize as "Secrets Manager" and refer to it only as `IBM Cloud Secrets Manager` on first use and `Secrets Manager` on subsequent use.

Also put periods (full stops) at end of each list item